login.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://xendesign.netlify.app/css/index.css">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://xendesign.netlify.app/js/index.js"></script>
+    <title>ZenCart</title>
+</head>
+
+<body class="bg-dark  overflow-x-hidden text-light">
+    <header
+        class="nav bg-dark-2 l-2 r-2 t-2  p-2 pl-3 pr-3 h-auto fixed row items-center z-50 br-lg b-solid b-1 h-min br-primary back-blur-5 bx-sh-primary-3">
+        <nav class="row items-center justify-between w-full ">
+            <ul class="left row items-center">
+                <h1 class="text-lg text-light">ZenCart</h1>
+            </ul>
+              <ul class="right row items-center flex-gap-1 text-light">
+                <li><a href="index.html">Home</a></li>
+                <li><button class="btn btn-light">
+                        <a href="login.html">
+                            Login</a></button></li>
```

Comment:
check alignment of tags

Reply:
I have used prettier extension still it's not fixing it.